base/workflow/R/start_model_runs.R

```diff
@@ -271,17 +279,10 @@ start_model_runs <- function(settings, write = TRUE, stop.on.error = TRUE) {
       }
       
       if (job_finished) {
-        
```

Comment:
this loop is needed since the file rabbitmq.out is generated on the remote side. see https://github.com/PecanProject/pecan/blob/master/docker/executor/executor.py#L88

One option is to only copy that one file (small) and at the end copy all files.

Reply:
What if we did this with `remote.execute.cmd()` instead?  I'm not actually sure which would be best for performance, but I suspect that would be more light weight than copying the log file.

Reply:
I'm not sure this is correct.  See line 265/273 above:
```
if (is_rabbitmq) {
        job_finished <- file.exists(file.path(jobids[run], "rabbitmq.out"))
```
This was always happening before the files get copied back over.  It's the same path referenced in L285/286, so it must be a file that exists before `remote.copy.from()` runs, right?